test/run-k8s-integration.sh

```diff
@@ -64,7 +66,7 @@ if [ "$deployment_strategy" = "gke" ]; then
     base_cmd="${base_cmd} --gke-cluster-version=${gke_cluster_version}"
   fi
 else
-  base_cmd="${base_cmd} --kube-version=${kube_version}"
+  base_cmd="${base_cmd} --kube-version=${kube_version} --kube-feature-gates=VolumeAttributesClass=true"
```

Comment:
Can this go after  --kube-runtime-config=api/all=true above?

Reply:
Resolved offline